examples/seismic/skew_self_adjoint/operators.py

```diff
@@ -0,0 +1,333 @@
+# TODO figure out if can get time sampling from the SparseTimeFuntions src/rec
```

Comment:
You mean resample it?  src/rec should have the  `.resample` utility for that.

Reply:
nvm